README.md

```diff
@@ -1,181 +1,196 @@
+# LTI 1.3 Advantage Library
```

Comment:
This is nit-picking, but I don't see "Advantage" mentioned anywhere else in the project. Seems kind of random for the main page title.

Reply:
LTI 1.3's full name is "LTI v1.3 and LTI Advantage" ... I'm not exactly sure what the terms mean, tbh. I think the "Advantage" part may refer to things like roster pull and gradebook sync, whereas the "1.3" part maybe refers to the actual launch process?